eZ/Bundle/EzPublishCoreBundle/Tests/DependencyInjection/Configuration/Parser/CommonTest.php

```diff
@@ -48,7 +46,7 @@ public function testIndexPage()
 
         $this->assertConfigResolverParameterValue('index_page', $indexPage1, 'ezdemo_site');
         $this->assertConfigResolverParameterValue('index_page', $indexPage2, 'ezdemo_site_admin');
-        $this->assertConfigResolverParameterValue('index_page', null, self::EMPTY_SA_GROUP);
+        $this->assertConfigResolverParameterValue('index_page', null, AbstractParserTestCase::EMPTY_SA_GROUP);
```

Comment:
There is no need to change `self::EMPTY_SA_GROUP` to `AbstractParserTestCase::EMPTY_SA_GROUP`